### PR TITLE
Update QC2 sample example

### PR DESCRIPTION
**Pull request by mwalzer**
Includes file validation fixes and explanation corrections

### CODE REVIEW

File: specification_documents/examples/intro_qc2.mzQC [hunk 1]
```diff
@@ -11,7 +11,7 @@
           "inputFiles": [
             {
               "location": "file://tmp/QC2_18052020.mzML",
-              "name": "QC type 2 sample",
+              "name": "QC2_18052020",
```
Comment (bittremieux) [inline specification_documents/examples/intro_qc2.mzQC:14]: **question:** Can the raw file and the ID file both have the same `name` attribute?

File: specification_documents/examples/intro_qc2.mzQC [hunk 2]
```diff
@@ -33,20 +33,30 @@
                   "value": "LTQ Orbitrap Velos"
                 }
               ]
+            },
+            {
+              "location": "file://tmp/QC2_18052020.mzid",
+              "name": "QC2_18052020",
+              "fileFormat": {
+                "accession": "MS:1002073",
+                "name": "mzIdentML format"
```
Comment (bittremieux) [inline specification_documents/examples/intro_qc2.mzQC:42]: **issue:** The mzIdentML currently doesn't seem to be recognized by the validator as a relevant ID file for ID-based metrics:

> ID based metric but no ID input file of severity 6 and message: ID based metrics present but no ID input file could be found registered in the mzQC file: run/set label =

File: specification_documents/examples/intro_qc2.mzQC [hunk 3]
```diff
@@ -33,20 +33,30 @@
...
+              }
             }
           ],
           "analysisSoftware": [
             {
               "accession": "MS:1001058",
               "name": "quality estimation by manual validation",
```
Comment (bittremieux) [inline specification_documents/examples/intro_qc2.mzQC:49]: **question:** Should the analysis software be manual validation and custom tool (currently) or a new CV term for QCloud?

File: specification_documents/examples/intro_qc2.mzQC [hunk 4]
```diff
@@ -55,26 +65,47 @@
           {
             "accession": "MS:4000060",
             "name": "number of MS2 spectra",
-            "value": 62299
+            "description": "The number of MS2 events in the run.",
+            "value": 62299,
+            "unit": {
```
Comment (bittremieux) [inline specification_documents/examples/intro_qc2.mzQC:70]: **issue:** These next 4 fours actually don't have a unit defined in the PSI-MS CV (they also predate the QC terms). Two related issues:
1. The validator still complains that a unit is missing, even though no unit is defined. This indicates a bug in the validator.
2. Should we propose to add units to pre-existing CV terms outside of the QC namespace?

Comment (mwalzer) [reply specification_documents/examples/intro_qc2.mzQC:70]: see https://github.com/MS-Quality-Hub/pymzqc/issues/47

Comment (bittremieux) [reply specification_documents/examples/intro_qc2.mzQC:70]: Units added in [psi-ms-CV#318](https://github.com/HUPO-PSI/psi-ms-CV/pull/318).

File: specification_documents/examples/intro_qc2.mzQC [hunk 5]
```diff
@@ -142,8 +174,8 @@
     "controlledVocabularies": [
       {
         "name": "Proteomics Standards Initiative Mass Spectrometry Ontology",
-        "uri": "https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.79/psi-ms.obo",
-        "version": "4.1.79"
+        "uri": "https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.157/psi-ms.obo",
+        "version": "4.1.157"
```
Comment (bittremieux) [inline specification_documents/examples/intro_qc2.mzQC:178]: **todo:** Version number needs to be updated after [psi-ms-CV#318](https://github.com/HUPO-PSI/psi-ms-CV/pull/318) has been merged.

**Review by bittremieux — APPROVED** [review]

**Pull request merged**